diff --git a/test-suite/Makefile.in b/test-suite/Makefile.in
index b7751e0..d959fd9 100644
--- a/test-suite/Makefile.in
+++ b/test-suite/Makefile.in
@@ -1985,7 +1985,6 @@ SCM_TESTS = tests/00-initial-env.test		\
 	    tests/popen.test			\
 	    tests/popen-child.scm		\
 	    tests/ports.test			\
-	    tests/posix.test			\
 	    tests/q.test			\
 	    tests/r4rs.test			\
 	    tests/r5rs_pitfall.test		\
